app/services/teacher_training_public_api/sync_courses.rb

```diff
@@ -32,7 +33,11 @@ def create_or_update_course(course_from_api, recruitment_cycle_year)
       add_accredited_provider(course, course_from_api[:accredited_body_code], recruitment_cycle_year)
       course.save!
 
-      TeacherTrainingPublicAPI::SyncSites.perform_async(provider.id, recruitment_cycle_year, course.id)
+      if @run_in_background
```

Comment:
can you add `run_in_background` to `attr_reader`?

Reply:
sure